pkg/tecdsa/signing/gen/pb/message.proto

```diff
@@ -20,4 +20,10 @@ message TSSRoundTwoMessage {
     uint32 senderID = 1;
     map<uint32, bytes> peersPayload = 2;
     string sessionID = 3;
+}
+
+message TSSRoundThreeMessage {
+    uint32 senderID = 1;
+    bytes payload = 2;
```

Comment:
`broadcastPayload`?

Reply:
Hmm... I think we were using `payload` and `broadcastPayload` pretty inconsistently unless this is not the same type of a payload:
- signing round one: `broadcastPayload`
- signing round three: `payload`
- key generation round one: `payload`
- key generation round two: `broadcastPayload`
- key generation round three: `payload`

Reply:
Basically, I use `payload` if there is only one type of payload in the given message and add the `peers*` and `broadcast*` prefixes if there are more types. I can change that to `broadcastPayload` even in the case of a single type.

Reply:
Yeah, I think we should use `broadcastPayload` everywhere. This is easier to reason about.

Reply:
Addressed by #3271